clojure/donkey/project.clj

```diff
@@ -0,0 +1,8 @@
+(defproject the-benchmarker-donkey "0.1.0-SNAPSHOT"
+  :dependencies [[org.clojure/clojure "1.10.1"]
+                 [com.appsflyer/donkey "0.4.0"]]
```

Comment:
is it required to pin the exactas  version in `clojure` ? could not we use a range, like `[0.4,0.5)` @yaronel

Reply:
Not sure how leiningen handles ranges. Will investigate.